app/src/main/kotlin/ReversiMain.kt

```diff
@@ -1,7 +1,112 @@
 package me.john_
 
+import GridWorld.GridWorldGridSolve
```

Comment:
Why is GridWorld code in the ReversiMain ?

Reply:
Ah good point, I should move the gridworld benchmarking code to its own main class.